docs/NativeModulesAndroid.md

```diff
@@ -89,21 +89,43 @@ ReadableArray -> Array
 ### Register the Module
 
 The last step within Java is to register the Module; this happens in the `createNativeModules` of your apps package. If a module is not registered it will not be available from JavaScript.
+In the package class it is important you override three methods: `createNativeModules`, `createJSModules` and `createViewManagers`.
+
+Note: It is necessary to have same package name of `MainActivity.java` in your packaging class and module class. Only then your module will be accessible across classes. 
```

Comment:
Sorry I noticed just now I don't understand this paragraph at all. By packaging class (should be package) do you mean `AnExampleReactPackage`? 

There is `MainActivity.java` in `AnExampleReactPackage`.